crypto/blake2/blake2s.c

```diff
@@ -14,10 +14,10 @@
  * can be found at https://blake2.net.
  */
 
+#include "e_os.h"
```

Comment:
If it's not needed in blake2b, there is no way it's needed in blake2s.

Reply:
Okay, I'll check this out.